docs/operator-manual/user-management/microsoft.md

```diff
@@ -176,3 +208,41 @@ data:
           groups:
             - DevOps
 ```
+
+## Validation
+
+### Log in to ArgoCD UI using SSO
+
+1. Open a new browser tab and enter your ArgoCD URI: https://`<my-argo-cd-url>`
+
+   ![Azure SSO Web Log In](../../assets/azure-sso-web-log-in-via-azure.png "Azure SSO Web Log In")
+
+3. Click `LOGIN VIA AZURE` button to log in with your Azure Active Directory account. You’ll see the ArgoCD applications screen.
+
+   ![Azure SSO Web Application](../../assets/azure-sso-web-application.png "Azure SSO Web Application")
+  
+4. Navigate to User Info and verify Group ID. Groups will have your group’s Object ID that you added in the `Setup permissions for Azure AD Application` step.
+
+   ![Azure SSO Web User Info](../../assets/azure-sso-web-user-info.png "Azure SSO Web User Info")
+
+### Log in to ArgoCD using CLI
+
+1. Open terminal, execute the below command.
+```
+argocd login <my-argo-cd-url> --grpc-web-root-path / --sso
+```
+2. You will see the below message after entering your credentials from the browser.
+
+   ![Azure SSO CLI Log In](../../assets/azure-sso-cli-log-in-success.png "Azure SSO CLI Log In")
+
+3. Your terminal output will be similar as below.
+
+```
+WARNING: server certificate had error: x509: certificate is valid for ingress.local, not my-argo-cd-url. Proceed insecurely (y/n)? y
```

Comment:
That warning might scare some people. Can we add some context on what it means and the effect of answering yes? It is harmless?